src/KafkaFlow/KafkaBus.cs

```diff
@@ -73,5 +80,17 @@ public Task StopAsync()
 
             return Task.WhenAll(this.consumerManagers.Select(x => x.StopAsync()));
         }
+
+        private async Task CreateMissingClusterTopics(ClusterConfiguration cluster)
+        {
+            if (cluster.TopicsToCreateIfNotExist is { Count: 0 })
+            {
+                return;
+            }
+
+            await this.topicManager.CreateIfNotExistsAsync(
+                string.Join(",", cluster.Brokers),
+                cluster.TopicsToCreateIfNotExist);
```

Comment:
I think we can pass the `cluster.Brokers` in the TopicManager ctor as is and apply the `string.Join()` there to create the AdminClient

Reply:
✅